demo/demo_app.py

```diff
@@ -10,7 +10,7 @@
 import pycyphal
 
 # DSDL files are automatically compiled by pycyphal import hook from sources pointed by CYPHAL_PATH env variable.
-import sirius_cyber_corp  # This is our vendor-specific root namespace. Custom data types.
+import arasaka_cyber_corp  # This is our vendor-specific root namespace. Custom data types.
```

Comment:
I prefer not to change this as there are references to this namespace on the forum that are better left valid. Also, I prefer my reference to the Hitchiker's Guide to the Galaxy to your reference to Cyberpunk.

Reply:
Never finished reading that book. :)

Cyberpunk is a lot cooler but ok.